.github/workflows/node.js.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Node.js setup
         uses: actions/setup-node@v4
         with:
-          node-version: '16.x'
+          node-version: '18'
```

Comment:
is it possible to update this to use:
```
node-version-file: .tool-versions
```

that way we don't need to update this in future upgrades?

Reply:
That is a good move!